helloworld/client.go

```diff
@@ -0,0 +1,94 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Runs a simple client, designed to interact with a personality.
+package helloworld
+
+import (
+	"context"
+
+	trillian "github.com/google/trillian"
+	"github.com/google/trillian/merkle/logverifier"
+	"github.com/google/trillian/merkle/rfc6962/hasher"
+
+	p "github.com/google/trillian-examples/helloworld/personality"
+)
+
+// Personality is the interface to the the Trillian log.
```

Comment:
nit: this comment is technically true at some level of abstraction, but given that a goal of this example is to clarify the actors I think it's a bit confusing.